tests/test-images/Dockerfile.2

```diff
@@ -1,4 +0,0 @@
-FROM scratch
```

Comment:
docker doesn't allow saving empty images.
You can take this Dockerfile (the one I deleted), place it in `/tmp` and build an image.
```
a) docker build -t empty /tmp
b) docker save empty > empty_image.tar
```
When trying (b) you should get an error `Error response from daemon: empty export - not implemented`. 

This is something docker doesn't provide. It has nothing to do with `atomic migrate export`